broadcast/errors/errors.go

```diff
@@ -0,0 +1,57 @@
+package broadcastErrors
+
+type BroadcastIDErr struct{}
+
+func (err BroadcastIDErr) Error() string {
+	return "wrong broadcastID"
+}
+
+type MissingClientReqErr struct{}
+
+func (err MissingClientReqErr) Error() string {
+	return "has not received client req yet"
+}
+
+type AlreadyProcessedErr struct{}
+
+func (err AlreadyProcessedErr) Error() string {
+	return "already processed request"
+}
+
+type ReqFinishedErr struct{}
+
+func (err ReqFinishedErr) Error() string {
+	return "request has terminated"
+}
+
+type ClientReqAlreadyReceivedErr struct{}
+
+func (err ClientReqAlreadyReceivedErr) Error() string {
+	return "the client req has already been received. The forward req is thus dropped."
+}
+
+type MissingReqErr struct{}
+
+func (err MissingReqErr) Error() string {
+	return "a request has not been created yet."
```

Comment:
Not sure I understand when this could happen, but here is a shorter error message:
```go
// Add comment here with more context
return "request not created yet"
```
Don't use `.` in error messages, since they often get chained:

```text
failed to update user: malformed request
```